src/hook.cc

```diff
@@ -991,7 +991,9 @@ parse(const char *func, void *address, unsigned *patches)
     //ENDBR64
     else if (insns[0] == 0xf3 && insns[1] == 0x0f && insns[2] == 0x1e && insns[3] == 0xfa )
       n +=4, insns += 4;
-
+    //MOVQ with XMM registers.
+    else if (insns[0] == 0x66 && insns[1] == 0x48 && insns[2] == 0xf)
```

Comment:
you probably should copy the remaining bytes in insns[3] and insns[4]

Reply:
The only mention I see is
https://github.com/igprof/igprof/blob/58942b67bc2b35a5c8e38024e322b8d3d28800c8/src/hook.cc#L1177

Reply:
indeed maybe it's not needed.